tests/_support/data/project/src/CodeSniffer/success/Success.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Acceptance\Extension\Tasks\CodeSniffer;
+
+class Success
+{
+    /**
+     * @var array<string>
+     */
+    protected array $test;
```

Comment:
According to type $test should be renamed to $tests

Reply:
fixed